lib/services/keyVault/README.md

```diff
@@ -1,8 +1,14 @@
 # Microsoft Azure SDK for Node.js - Key Vault
 
-This project provides a Node.js package for accessing keys, secrets and certificates on Azure Key Vault. Right now it supports:
+This project provides a Node.js package for accessing keys, secrets and certificates on Azure Key Vault.
+
+Right now it supports:
 - **Node.js version: 6.x.x or higher**
 - **REST API version: 7.0**
+- **ms-rest-azure version 2.6.0**.  
+  We recommend authenticating with `KeyvaultCredentials`, either the one exported by this package (`azure-keyvault`),
```

Comment:
What does `KeyvaultCredentials` provide that other login credentials from `azure-keyvault` do not have?

Reply:
I could be understanding what you said wrongly. I am trying to express this what you said here https://github.com/Azure/azure-sdk-for-node/issues/5153#issuecomment-636248372 :

> update readme with a note on when KeyvaultCredentials should be used instead of the credentials from ms rest azure

Reply:
My bad, I should have been more clear.

From your investigations, what I have understood is that both KeyvaultCredentials and the normal credentials from ms-rest-azure work with azure-keyvault package.

As a user, it is not clear to me as to why or when should I use KeyvaultCredentials vs the other normal credentials from ms-rest-azure. What is the difference between the two? Which of the two are we advocating for?

Reply:
It is as you said, any of them work as long as we're in 2.6.0. I don't personally think we should suggest using one authentication method over the other, though I do think we need to mention the 2.6.0 requirement. I will fix my changes to express what I'm stating in this comment.

Reply:
I mean, I think it would be fair to say only this then?

> Right now it supports:
> - **Node.js version: 6.x.x or higher**
> - **REST API version: 7.0**
> - **ms-rest-azure version 2.6.0**.

Reply:
I believe it might need a little more attention than clubbing with node version which people usually skip over

Reply:
We can handle the note on `KeyvaultCredentials` in a separate PR if needed